src/Illuminate/Queue/WorkerOptions.php

```diff
@@ -73,7 +87,8 @@ class WorkerOptions
      * @param  bool  $stopWhenEmpty
      * @return void
      */
-    public function __construct($name = 'default', $backoff = 0, $memory = 128, $timeout = 60, $sleep = 3, $maxTries = 1, $force = false, $stopWhenEmpty = false)
+    public function __construct($name = 'default', $backoff = 0, $memory = 128, $timeout = 60, $sleep = 3, $maxTries = 1,
```

Comment:
The added params should be also added to the docblock.

Reply:
@brunogaspar think Mohamed did that? To which params are you referring?

Reply:
He did it just now, see the commits.

Reply:
Ah I see 👍